google-cloud-bigquerystorage/clirr-ignored-differences.xml

```diff
@@ -212,5 +212,21 @@
     <className>com/google/cloud/bigquery/storage/v1/StreamWriter</className>
     <method>void setMissingValueInterpretationMap(java.util.Map)</method>
   </difference>
+  <difference>
+    <!--This class is marked as @InternalApi "public for technical reasons"-->
+    <differenceType>6004</differenceType>
+    <className>com/google/cloud/bigquery/storage/*/stub/readrows/ApiResultRetryAlgorithm</className>
+    <field>DEADLINE_SLEEP_DURATION</field>
+    <from>org.threeten.bp.Duration</from>
+    <to>java.time.Duration</to>
+  </difference>
```

Comment:
I don't think I see this in the PR. Is this an old implementation?

Reply:
It doesn't show up as a direct change because it's declared as `Duration DEADLINE_SLEEP_DURATION`. The change in the file was done in the [import statement](https://github.com/googleapis/java-bigquerystorage/pull/2780/files#diff-6fed86adbffd5ca2037793806c832765de4b7b04dbcc85b325b93d61b9f3c33f) so the type of this internal var still changes.

Reply:
Ok, I see the class is marked with InternalApi. I think this is fine